server.js

```diff
@@ -680,8 +680,11 @@ app.post('/submit-match',  asyncMiddleware( async (req, res, next) => {
                     pending_matches
                       .filter(e => ((e.network1 === req.body.winnerhash && e.network2 === req.body.loserhash) ||
```

Comment:
Do you need ((( at the start of this?

Reply:
Maybe not. I'm just not used to using callbacks without a { } wrapper.

Reply:
{} is only needed for multi line blocks. If it's a single expression you do not need it. Same as with `if(b) return;` vs `if(b) { i++; return; }`.